TicketingService.Monitoring/Handlers.cs

```diff
@@ -0,0 +1,53 @@
+using Marten;
+using Newtonsoft.Json;
+using TicketingService.Abstractions;
+
+namespace TicketingService.Monitoring
+{
+    public static class Handlers
+    {
+        private static DateTimeOffset _fromDate = new DateTimeOffset(new DateTime(2022, 10, 20), TimeSpan.Zero);
+
+        public static async Task<string> GetDistribution(IDocumentStore store, CancellationToken cancellationToken)
+        {
+            var completedTickets = await store.GetCompletedTickets(cancellationToken);
+
+            var completionTimes = completedTickets
+                .Select(t => t.LastModified.Subtract(t.Created))
+                .OrderBy(t => t);
+
+            var groupCount = 10;
+            var groupSize = Math.Ceiling(completionTimes.Max().TotalSeconds / groupCount);
```

Comment:
If 1 ticket takes a very long time, the distribution will be like group 1: all - 1, group 2-9: 0, group 10: 1.
I think it would make sense to have a default of ranges or have like `P99, P95, P90, ....`